test/extended/oauth/client_secret.go

```diff
@@ -113,6 +116,31 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer]", func() {
 	})
 })
 
+func getOAuthEndpoint(apiserverURL string) (string, error) {
```

Comment:
this could be replaced by https://github.com/openshift/origin/blob/5baea95eac9b776d483ca2ba89f6eca9e264e80d/test/extended/oauth/helpers.go#L44-L54

Reply:
perfect, updated it to use that